tests/Label/ReadModels/Relations/ProjectorTest.php

```diff
@@ -476,7 +476,7 @@ private function getRelationId()
         return 'E4CA9DB5-DEE3-42F0-B04A-547DFC3CB2EE';
     }
 
-    private function createDomainMessage(string $id, Serializable $payload): DomainMessage
+    private function createDomainMessage($id, SerializableInterface $payload)
```

Comment:
Why did you remove the `string` param type and `DomainMessage` return type again here?

Reply:
The `DomainMessage` return type was re-added in a later commit I see, but the `string` is still gone from the final diff

Reply:
That was indeed a mistake because of doing partial staging to have better commits. It fixed it.